src/OpenLoco/Windows/News/News.h

```diff
@@ -8,7 +8,6 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Ui::Windows::NewsWindow
 {
-    static loco_global<uint8_t[3], 0x005215B5> _unk_5215B5;
```

Comment:
for reference 5215B5 is an empty string used for priming the text colour global var.